doc/source/development.rst

```diff
@@ -29,8 +29,11 @@ recompile much more quickly by doing
 
 .. code-block:: shell
 
- cd ray/build
- make -j8
+ cd ray
+ bazel build //:ray_pkg
+
+This command is not enough to recompile all C++ unit tests. To do so, see
+:ref:`Testing locally`.
```

Comment:
Did you test that this link works as expected when you build the documentation?

Reply:
I tested it (it needed to be slightly changed)

Reply:
Thanks!